app/gui2/src/components/GraphNode.vue

```diff
@@ -22,7 +22,7 @@ const props = defineProps<{
 const emit = defineEmits<{
   updateRect: [rect: Rect]
   updateExprRect: [id: ExprId, rect: Rect]
-  updateContent: [range: ContentRange, content: string]
+  updateContent: [updates: [ContentRange, string][]]
```

Comment:
```suggestion
  updateContent: [updates: [range: ContentRange, content: string][]]
```
alternatively it might be a good idea to make it an object type (potentially with a separate type definition) instead